src/EnergyPlus/DuctLoss.cc

```diff
@@ -0,0 +1,1084 @@
+                                Constant::Units::C,
+                                state.afn->AirflowNetworkNodeSimu(NodeNum).TZ,
+                                OutputProcessor::TimeStepType::System,
+                                OutputProcessor::StoreType::Average,
+                                state.afn->AirflowNetworkNodeData(NodeNum).Name);
+            SetupOutputVariable(state,
+                                "Duct Loss Node Humidity Ratio",
+                                Constant::Units::kgWater_kgDryAir,
+                                state.afn->AirflowNetworkNodeSimu(NodeNum).WZ,
+                                OutputProcessor::TimeStepType::System,
+                                OutputProcessor::StoreType::Average,
+                                state.afn->AirflowNetworkNodeData(NodeNum).Name);
+        }
+
+        for (int DuctLossNum = 1; DuctLossNum <= state.dataDuctLoss->NumOfDuctLosses; DuctLossNum++) {
+            SetupOutputVariable(state,
+                                "Duct Loss Sensible Loss Rate",
+                                Constant::Units::W,
+                                state.dataDuctLoss->ductloss(DuctLossNum).Qsen,
+                                OutputProcessor::TimeStepType::System,
+                                OutputProcessor::StoreType::Average,
+                                state.dataDuctLoss->ductloss(DuctLossNum).Name);
+            SetupOutputVariable(state,
+                                "Duct Loss Latent Loss Rate",
+                                Constant::Units::kgWater_s,
+                                state.dataDuctLoss->ductloss(DuctLossNum).Qlat,
+                                OutputProcessor::TimeStepType::System,
+                                OutputProcessor::StoreType::Average,
+                                state.dataDuctLoss->ductloss(DuctLossNum).Name);
+        }
+
+        for (int ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ZoneNum++) {
+            SetupOutputVariable(state,
+                                "Zone Added Sensible Rate Due to Duct Loss",
+                                Constant::Units::W,
+                                state.dataDuctLoss->ZoneSen(ZoneNum),
+                                OutputProcessor::TimeStepType::System,
+                                OutputProcessor::StoreType::Average,
+                                state.dataHeatBal->Zone(ZoneNum).Name);
+            SetupOutputVariable(state,
+                                "Zone Added Latent Rate Due to Duct Loss",
+                                Constant::Units::kgWater_s,
+                                state.dataDuctLoss->ZoneLat(ZoneNum),
+                                OutputProcessor::TimeStepType::System,
+                                OutputProcessor::StoreType::Average,
+                                state.dataHeatBal->Zone(ZoneNum).Name);
+        }
+
+        if (state.dataDuctLoss->NumOfDuctLosses > 0) {
```

Comment:
Noticed RDD diffs on many test files:
```
+Output:Variable,*,Zone Added Latent Rate Due to Duct Loss,hourly; !- HVAC Average [W]
+Output:Variable,*,Zone Added Sensible Rate Due to Duct Loss,hourly; !- HVAC Average [W]
```
These shouldn't be created if these are no duct loss objects.
Move this `if` up higher? Before the `SetupOutputVariable` calls and maybe more. Move up to line 348?